develop/parachains/deployment/coretime-renewal.md

```diff
@@ -0,0 +1,257 @@
+
+- [`enable_auto_renew(core, task, workload_end_hint)`](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/pallet/struct.Pallet.html#method.enable_auto_renew){target=\_blank} - use this extrinsic to activate automatic renewals for a specific core. This transaction must originate from the sovereign account of the parachain task
+
+    **Parameters:**
+
+    - **`core`** - the core currently assigned to the task
+    - **`task`** - the task for which auto-renewal is being enabled
+    - **`workload_end_hint`** - the timeslice at which the currently assigned core will stop being used. This value helps the system determine when auto-renewal should begin. It is recommended to always provide this value to avoid ambiguity
+
+        - If the coretime expires in the current sale period, use the last timeslice of the current sale period
+
+        - If the coretime expires at the end of the next sale period (e.g., because you've already renewed), use the last timeslice of the next sale period
+
+        - If a lease is active, use the timeslice when the lease ends
+
+- [`disable_auto_renew(core, task)`](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/pallet/struct.Pallet.html#method.disable_auto_renew){target=\_blank} - use this extrinsic to stop automatic renewals. This extrinsic also requires that the origin is the sovereign account of the parachain task
+
+     **Parameters:**
+
+    - **`core`**: the core currently assigned to the task
+    - **`task`**: the task for which auto-renewal is enabled
+
+### Construct the Enable Auto Renewal Extrinsic
+
+To configure auto-renewal, you'll need to gather specific information for the `enable_auto_renew` extrinsic parameters:
+
+- **`core`** - identify which core your parachain is assigned to when the it expires. This requires checking both current assignments and planned future assignments:
+    - For current period - query `broker.workload()`
+    - For next period - query `broker.workplan()`
+
+    **Example for parachain `2000`:**
+    
+    Current assignment (workload)
+    ```
+    [
+      [50]
+      [{
+        mask: 0xffffffffffffffffffff
+        assignment: {Task: 2,000}
+      }]
+    ]
+    ```
+
+    Future assignment (workplan)
+    ```
+    [
+      [[322,845, 48]]
+      [{
+        mask: 0xffffffffffffffffffff
+        assignment: {Task: 2,000}
+      }]
+    ]
+    ```
+
+    **Note:** use the core from workplan (`48` in this example) if your task appears there. Only use the core from workload if it's not listed in workplan.
+
+- **`task`** - use your parachain ID, which can be verified by connecting to your parachain and querying `parachainInfo.parachainId()`
+
+- **`workload_end_hint`** - you should always set it explicitly to avoid misbehavior. This value indicate when your assigned core will expire. Here's how to calculate the correct value based on how your core is assigned:
```

Comment:
```suggestion
- **`workload_end_hint`** - you should always set it explicitly to avoid misbehavior. This value indicates when your assigned core will expire. Here's how to calculate the correct value based on how your core is assigned:
```